pkg/eks/eks.go

```diff
@@ -82,6 +83,14 @@ func (c *ClusterProvider) SupportsManagedNodes(clusterConfig *api.ClusterConfig)
 
 // ClusterSupportsManagedNodes reports whether the EKS cluster supports managed nodes
 func ClusterSupportsManagedNodes(cluster *awseks.Cluster) (bool, error) {
+	supportsManagedNodes, err := utils.IsMinVersion(api.Version1_15, *cluster.Version)
```

Comment:
should the minimum version for managed groups be `api.Version1_14` instead of 
`utils.IsMinVersion(api.Version1_15, *cluster.Version)`

Reply:
The minimum required version is 1.14 but 1.14 requires a platform version check also. Both of this happen below in line 94. This check is for returning early to avoid doing a platform version check.